src/game/common/system/quotedprintable.cpp

```diff
@@ -49,3 +49,47 @@ Utf8String Ascii_String_To_Quoted_Printable(Utf8String string)
     dest[count] = '\0';
     return dest;
 }
+
+int Hex_Digit_To_Int(char c)
+{
+    if (c <= '9' && c >= '0') {
+        return c - '0';
+    }
+
+    if (c <= 'f' && c >= 'a') {
+        return c - 'W';
+    }
+
+    if (c > 'F' || c < 'A') {
+        return 0;
+    }
+    return c - '7';
+}
+
+Utf8String Quoted_Printable_To_Ascii_String(Utf8String string)
+{
+    static char dest[1024];
+    char *buf = dest;
+
+    for (const char *c = string.Str(); *c != '\0'; c++) {
+        if (*c == '_') {
+            if (c[1] == '\0') {
+                break;
+            }
+
+            *buf = Hex_Digit_To_Int(*++c);
+
+            if (c[1] != '\0') {
+                *buf *= 16;
+                *buf = Hex_Digit_To_Int(*++c) | *buf;
+            }
+        } else {
+            *buf = *c;
```

Comment:
Might as well write `c[0]` here for consistency. Already uses `c[1]` in this function.

Reply:
fixed.